### ISSUE TITLE
camera faces ground

### ISSUE THREAD

**Issue by ChristophTobler** [PX4/PX4-SITL_gazebo-classic#107]
@LorenzMeier Your [last commit](https://github.com/PX4/sitl_gazebo/commit/622efea2f29be7dc5da7494769237aacf5b625ba) is not actually working for me
The output is:
```
[Err] [Camera.cc:1700] Unable to attach to visual with name[iris]
```
resp.
```
[Err] [Camera.cc:1700] Unable to attach to visual with name[iris_opt_flow]
```
![iris_camera](https://user-images.githubusercontent.com/14769636/27793391-1ec8b62a-5ffe-11e7-9cfa-1c16a06d5910.png)
![iris_opt_flow_camera](https://user-images.githubusercontent.com/14769636/27793399-2482547c-5ffe-11e7-8822-d6a59f229dd9.png)

**Comment by LorenzMeier** [comment]
It is working in Gazebo 8 - what version do you have?

**Comment by ChristophTobler** [comment]
Gazebo 7, like probably most Ubuntu/ROS Kinetic users.

**Comment by LorenzMeier** [comment]
Fixed in 9bb3ab4a7 in Firmware and here in master.
